src/deprecated/deprecated.js

```diff
@@ -572,6 +579,63 @@ GraphicsDevice.prototype.removeShaderFromCache = function (shader) {
     getProgramLibrary(this).removeFromCache(shader);
 };
 
+const _tempBlendState = new BlendState();
+
+GraphicsDevice.prototype.setBlendFunction = function (blendSrc, blendDst) {
```

Comment:
slightly concerned about performance impact on existing code setting a bunch of states, but not sure there's much we can do about it.

Reply:
yeah internally in the engine I removed all those, and print deprecated message.
The cost would not be that large though I think.